ivy_tests/test_docstrings.py

```diff
@@ -172,15 +280,31 @@ def check_docstring_examples_run(
 
     # assert output == parsed_output, "Output is unequal to the docstrings output."
     sig_fig = float("1e-" + str(num_sig_fig))
+    atol = sig_fig / 10000
     numeric_pattern = re.compile(
         r"""
-            [\{\}\(\)\[\]]|\w+:
+            [\{\}\(\)\[\]\<>]|\w+:
         """,
         re.VERBOSE,
     )
+
+    # word list to remove from execution output
+    words_to_remove = [
```

Comment:
not sure why we'd have an example that generates a non-`ivy` result (I mean `ivy.NativeArray`, `ivy.NativeShape`, etc. like `torch.Size`, `tf.Tensor`). The docstring examples should ideally generate `ivy`-specific objects (`ivy.Array`, `ivy.Shape`, etc.). So ideally there shouldn't be anything in the list `words_to_remove` except `ivy.array` and `ivy.Shape`, happy to know if there are any corner cases.

Reply:
Below ones fails when I remove that list and only trim "ivy.shape" and "ivy.array".: 
vmap
slogdet
svdvals

I believe this there are few more hidden cases. Good thing about having a list like that save "slogdet" test with a single element in the list. Some of them can be easily fixed by adding one word to that list.

```
output for  vmap  on run:  torch.Size([3,5,2]) 
output in docs : (3,5,2) 
 torch.Size3  !=  3 

output for  slogdet  on run:  slogdet(sign=ivy.array(1.),logabsdet=ivy.array(1.60943794)),slogdet(sign=ivy.array(-1.),logabsdet=ivy.array(1.0986129)),[{a:ivy.array(-1.),b:ivy.array(-1.)},{a:ivy.array(0.69314718),b:ivy.array(1.09861231)}] 
output in docs : slogdet(sign=ivy.array(1.),logabsdet=ivy.array(1.60943794)),slogdet(sign=ivy.array(-1.),logabsdet=ivy.array(1.098611)),[{a:ivy.array(-1.),b:ivy.array(-1.)},{a:ivy.array(0.69314718),b:ivy.array(1.09861231)}] 
 logabsdet=1.0986129  !=  logabsdet=1.098611 

output for  svdvals  on run:  ivy.Shape(2,),ivy.Shape(2,),ivy.array([0.,0.]),torch.Size([4,3]),ivy.array([10.25994301,1.16403711,0.61529785]),ivy.array([10.25994301,1.16403687,0.61529785]),ivy.array([0.39777088,0.15413129,0.1029852]),{a:ivy.array([9.01383495,0.86647373]),b:ivy.array([15.77865314,5.55970573,4.16857529,0.86412716])},ivy.array([13.37566662,3.88478065]),{a:ivy.array([12.95925522,4.64447212,2.54687858]),b:ivy.array([23.16135025,10.35037804,4.31025028,1.35769403])} 
output in docs : ivy.Shape(2,),ivy.Shape(2,),ivy.array([0.,0.]),(4,3),ivy.array([10.3,1.16,0.615]),ivy.array([10.3,1.16,0.615]),ivy.array([0.39777088,0.15413129,0.1029852]),{a:ivy.array([9.01383495,0.86647356]),b:ivy.array([15.7786541,5.55970621,4.16857576,0.86412698])},ivy.array([13.37566757,3.88477993]),{a:ivy.array([12.95925522,4.6444726,2.54687881]),b:ivy.array([23.16134834,10.35037804,4.31025076,1.35769391])} 
 torch.Size4  !=  4 
```

Reply:
I think we should be fixing the implementations of `vmap` and `svdvals` to ensure that native arrays and shapes aren't returned and we return `ivy` arrays and shapes instead (probably a fix to the function wrappers for the 2 functions). As for `slogdet`, we could either pick a simpler example in the docstring or update the tolerance for testing that function rather than skipping the tests for these functions entirely. 
Feel free to point out other functions where such issues exist, we should fix all of them on a case-by-case basis 😄

Reply:
Problem is not tolerance, since there are strings on there, numeric checks fails. When I just trim "logabsdet=" part, it passes tests. For fixing implementations, aren't those out of the scope of this PR? Maybe after merging this as it is, I can go one by one for failures require deeper investigation as another task or please guide about which wrappers to look at.